src/components/GameSpecBuilderContainer.js

```diff
@@ -202,13 +216,26 @@ class GameSpecBuilderContainer extends React.Component {
       }
 
       let value = {
-        spec: this.vars.spec,
-        uploader_uid: auth.currentUser.uid,
-        createdOn: firebase.database.ServerValue.TIMESTAMP
+        board: {
+          backgroundColor: 'FFFFFF',
+          imageId: this.state.selectedBoard,
+          maxScale: 1
+        },
+        gameIcon50x50: '-KwqEPnE2xzAON9V2mcP',
```

Comment:
Move all of the constant strings here to constants and use them by importing constants